yky-social/yky/app/components/UserSearch.tsx

```diff
@@ -23,21 +21,21 @@ const FollowButton : React.FC<FBProps> = ({uid, children }) => {
     const router = useRouter();
 
     const followClick = async () => {
-        const body = { action: "follow",  targetuser: uid };
+        const body : DoFollowRequest = { followUid: uid };
```

Comment:
I see a `DoFollow()` function in `DefaultApi.ts`, which sends this request for you. Can we use that instead?

Reply:
So, that's a follow-on.  This module is 'use client'.  The current use of OpenAPI is between the front end and backend, not from client to front end.  That said, using the data structure helps the compiler catch errors.  I'm thinking over the best way to go from client to frontend.